sleap/gui/app.py

```diff
@@ -325,8 +325,18 @@ def _create_video_player(self):
         self.player.seekbar.selectionChanged.connect(lambda: self.updateStatusMessage())
         self.setCentralWidget(self.player)
 
-        def switch_frame(video):
```

Comment:
`self` doesn't need to be passed in since this is an inner function. Inner functions will already have access to any variables defined before the inner function definition (such as `self`). The `video` however is not yet defined and constantly changing, so that is why we must pass in `video`. By adding `self`, the `Video` object is actually being passed in to the first positional argument `self` and nothing gets passed into `video`:

```python
Error occurred during callback 1 for video!
[<bound method QtVideoPlayer.load_video of <sleap.gui.widgets.video.QtVideoPlayer(0x23391a7cb70) at 0x00000233991182C8>>, <function MainWindow._create_video_player.<locals>.switch_frame at 0x0000023399139F78>, <function MainWindow._create_video_player.<locals>.<lambda> at 0x00000233993420D8>, <function MainWindow._create_video_player.<locals>.update_frame_chunk_suggestions at 0x0000023399342048>]
switch_frame() missing 1 required positional argument: 'video'
```